_includes/code/components/buttons.html

```diff
@@ -1,53 +1,19 @@
-<h6>Primary Buttons</h6>
+<h6>Primary buttons</h6>
 <div class="button_wrapper">
-  <button>Default</button>
-  <button class="usa-button-active">Active</button>
-  <button class="usa-button-hover">Hover</button>
-</div>
-<div class="button_wrapper">
-  <button class="usa-button-primary-alt">Default</button>
-  <button class="usa-button-primary-alt usa-button-active">Active</button>
-  <button class="usa-button-primary-alt usa-button-hover">Hover</button>
-</div>
-
-<h6>Secondary Buttons</h6>
-<div class="button_wrapper">
-  <button class="usa-button-secondary">Default</button>
-  <button class="usa-button-secondary usa-button-active">Active</button>
-  <button class="usa-button-secondary usa-button-hover">Hover</button>
-</div>
-
-<div class="button_wrapper">
-  <button class="usa-button-gray">Default</button>
-  <button class="usa-button-gray usa-button-active">Active</button>
-  <button class="usa-button-gray usa-button-hover">Hover</button>
+  {% fractal_component buttons--primary %}
 </div>
 
+<h6>Secondary buttons</h6>
 <div class="button_wrapper">
-  <button class="usa-button-outline" type="button">Default</button>
-  <button class="usa-button-outline usa-button-active">Active</button>
-  <button class="usa-button-outline usa-button-hover">Hover</button>
+{% fractal_component buttons--secondary %}
 </div>
 
+<h6>Buttons on a dark background</h6>
 <div class="button_wrapper button_wrapper-dark">
```

Comment:
currently don't need the `button_wrapper-dark` class since a dark bg is being applied from what's being generated on fractal